Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,113 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Cinemachine;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private float moveSpeed;
+    [SerializeField] private float jumpSpeed;
+
+    private float _distanceFromEarth = 0.5f;
+    private float _xVector;
+    private bool _moveable;
+    private Rigidbody2D _rigidbody2D;
+    private Animator _animator;
+    private CinemachineVirtualCamera _cvm;
+
+    private void OnEnable()
+    {
+        Timer.Instance.onFinishCountDown += EnableMoving;
+    }
+
+    private void OnDisable()
+    {
+        Timer.Instance.onFinishCountDown -= EnableMoving;
+    }
+
+    private void Awake()
+    {
+        _moveable = false;
+    }
+
+    private void Start()
+    {
+        _rigidbody2D = GetComponent<Rigidbody2D>();
+        _animator = GetComponent<Animator>();
+    }
+
+    private void Update()
+    {
+        Run();
+        Jump();
+        HandleAnimation();
+    }
+
+    private void Run()
+    {
+        if (_animator.GetBool("IsAlive") && _moveable)
+        {
+            _xVector = Input.GetAxisRaw("Horizontal");
+            transform.position += (Vector3)new Vector2(_xVector * moveSpeed * Time.deltaTime, 0);
+            Flip();
```

Comment:
Change this into a `ChangeScale(Vector2 value)` and remove the if inside the function